On The Map/Controller/LoginViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  LoginViewController.swift
+//  On The Map
+//
+//  Created by Michal Hus on 2/20/20.
+//  Copyright © 2020 Michal Hus. All rights reserved.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController {
+    
+    var students:[Student] = []
+    let loginErrorTitle:String = "Log In Failure"
```

Comment:
small style thing: you want a space between the : and the type name, like `students: [Student]` instead of `students:[Student]`